### PR TITLE
Do not set staging label when fetching the secret values.

### PR DESCRIPTION
**Pull request by ojecborec**

### CODE REVIEW

**Review by ryanjbaxter — CHANGES_REQUESTED** [review]
One final request, can you add some documentation on the new property to https://github.com/spring-cloud/spring-cloud-config/blob/4.0.x/docs/src/main/asciidoc/spring-cloud-config.adoc#aws-secrets-manager-backend

**Comment by ojecborec** [comment]
Sure. See my latest commit.

File: docs/src/main/asciidoc/spring-cloud-config.adoc [hunk 1]
```diff
@@ -1158,6 +1173,7 @@ AWS Secrets Manager API credentials are determined using link:https://docs.aws.a
 [NOTE]
 ====
 - When no application is specified `application` is the default, and when no profile is specified `default` is used.
+- Both `label` and `defaultLabel` properties are ignored, when `ignoreLabel` one is set to `true`.
```
Comment (ryanjbaxter) [inline docs/src/main/asciidoc/spring-cloud-config.adoc:1176]: I think you should remove the word `one`

File: docs/src/main/asciidoc/spring-cloud-config.adoc [hunk 2]
```diff
@@ -915,6 +915,21 @@ Note that if the default label is not set and a request does not define a label,
 
 Note that if the staging label contains a slash (`/`), then the label in the HTTP URL should instead be specified with the special string `({special-string})` (to avoid ambiguity with other URL paths) the same way <<_git_backend,Git backend's section>> describes it.
 
+Use `spring.cloud.config.server.aws-secretsmanager.ignore-label` property to ignore the `{label}` parameter of the HTTP resource as well as `spring.cloud.config.server.aws-secretsmanager.default-label` property. Repository will use secrets as if labelled version support is disabled.
```
Comment (ryanjbaxter) [inline docs/src/main/asciidoc/spring-cloud-config.adoc:918]: Add `The` before `Repository`

File: docs/src/main/asciidoc/spring-cloud-config.adoc (hunk 1 above)
Comment (ojecborec) [reply docs/src/main/asciidoc/spring-cloud-config.adoc]: Done.

File: docs/src/main/asciidoc/spring-cloud-config.adoc (hunk 2 above)
Comment (ojecborec) [reply docs/src/main/asciidoc/spring-cloud-config.adoc]: Done.

**Pull request merged**